qa/workunits/rbd/qemu_dynamic_features.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh -ex
+
+if [[ -z "${IMAGE_NAME}" ]]; then
+  echo image name must be provided
+  exit 1
+fi
+
+is_qemu_running() {
+  rbd status ${IMAGE_NAME} | grep -v "Watchers: none"
+}
+
+wait_for_qemu() {
+  while ! is_qemu_running ; do
+    echo "*** Waiting for QEMU"
+    sleep 30
+  done
+}
+
+wait_for_qemu
+while is_qemu_running ; do
+  echo "*** Disabling all features"
+  rbd feature disable ${IMAGE_NAME} journaling || true
```

Comment:
seems like we should check the return status when these are meant to succeed - maybe have an inital setup that gets the image to a known state with || true